app/controllers/bg_controller.rb

```diff
@@ -0,0 +1,7 @@
+class BgController < ApplicationController
```

Comment:
BgController ???????????????????????
i dont understand controller
if it is a blog controller name should be BlogController or BloggerController
dont use shortcut for Class names, method names, variable name etc
kindly strictly follow this naming convention